lib/lotus/commands/generate/abstract.rb

```diff
@@ -0,0 +1,75 @@
+require 'lotus/environment'
+require 'thor'
+
+module Lotus
+  module Commands
+    class Generate < Thor
+      class Abstract
```

Comment:
IMHO `Abstract` classes like this are better to not hold any logic, only the interface. Also, it's no good use inheritance to share code, [here are some reasons](https://www.youtube.com/watch?v=OMPfEXIlTVE) why it should be avoided.

Maybe we can pull all this test framework generation logic to a place that does specifically one thing as I did here #325